.github/workflows/trigger_release_creation.yml

```diff
@@ -28,13 +28,19 @@ jobs:
         run: npm ci
       - id: version-upgrade
         name: Upgrade Version
+        # Make next RC Final
         run: |
-          npm run release-make-next-rc-final
+          npm run npm version minor
```

Comment:
I think this should only be "npm version minor"

Reply:
Of course, thanks!